qa/rpc-tests/wallet.py

```diff
@@ -102,7 +117,7 @@ def run_test (self):
         assert_equal(self.nodes[2].getbalance(), 100)
         assert_equal(self.nodes[2].getbalance("from1"), 100-21)
 
-        # Send 10 BTC normal
+        print "Send 10 BTC normal"
```

Comment:
This was/is to help those (me) observe more easily where the sync 
process gets hung when running on windows.

On 20/08/2015 8:59 AM, Jonas Schnelli wrote:
>
> In qa/rpc-tests/wallet.py 
> <https://github.com/bitcoin/bitcoin/pull/6548#discussion_r37547012>:
>
> > @@ -102,7 +117,7 @@ def run_test (self):
> >          assert_equal(self.nodes[2].getbalance(), 100)
> >          assert_equal(self.nodes[2].getbalance("from1"), 100-21)
> >
> > -        # Send 10 BTC normal
> > +        print "Send 10 BTC normal"
>
> What's the reason of adding verbose test output to |wallet.py|?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/bitcoin/bitcoin/pull/6548/files#r37547012>.
>